docs/nilup.md

```diff
@@ -1,6 +1,30 @@
 # nilup
 
-`nilup` is the tool to manage Nillion SDK versions.
+`nilup` is an installer and version manager for the Nillion SDK
+
+To install `nilup`, run: `curl https://nilup.nilogy.xyz | bash`
+
+If you install the Nillion SDK using `nilup`, all the Nillion SDK commands will be managed by `nilup`
+and it will pick the version to use base on the next order.
+
+### `+<version>` flag
+When you call any Nillion SDK command like `nada` you can add a `+<version>` argument to use that version of `nada`.
+For example if you run `nada +0.0.4 init` will run `nada` version 0.0.4.
```

Comment:
```suggestion
When you run any Nillion SDK command, like `nada`, you can add a `+<version>` argument to use that version of the command.
For example, if you run `nada +0.0.4 init`, this will run `nada` version 0.0.4.
```

Reply:
how about this last suggestion? :)